processor/schematransformerprocessor/testdata/schema.yml

```diff
@@ -0,0 +1,110 @@
+# Defines the file format. MUST be set to 1.0.0.
+file_format: 1.0.0
+
+# The Schema URL that this file is published at. The version number in the URL
+# MUST match the highest version number in the "versions" section below.
+# Note: the schema version number in the URL is not related in any way to
+# the file_format setting above.
+schema_url: https://opentelemetry.io/schemas/1.1.0
+
+# Definitions for each schema version in this family.
+# Note: the ordering of versions is defined according to semver
+# version number ordering rules.
+versions:
+  1.1.0:
+    # Definitions for version 1.1.0.
+    all:
+      # Definitions that apply to all data types.
+      changes:
+        # Transformations to apply when converting from version 1.0.0 to 1.1.0.
+        - rename_attributes:
```

Comment:
That looks nice, where is this coming from? Would we be maintaining this here in this repo?

Reply:
This was the example taken from the specification https://github.com/open-telemetry/opentelemetry-specification/blob/e16e1a781bdf62a8fcfa1a12963844749de23e7a/specification/schemas/overview.md#collector-assisted-schema-transformation 

The idea here is to use this within future testing one the schema transformer is ready.

Reply:
As for who owns this schema translation files, I believe that is Instrumentation group led by @tedsuo

Reply:
The schemas themselves live in https://github.com/open-telemetry/opentelemetry-specification/tree/main/schemas

Reply:
Schema files are part of the Otel specification and are published at `https://opentelemetry.io/schemas/*` for every new specification release. The specification SIG owns these files.